File: LeetCode-Programs/Generate a String With Characters That Have Odd Counts.py
class Solution:
    def generateTheString(self, n: int) -> str:
        strr=""
        if n%2==0:
            for i in range(n-1):
                strr+='p'
            strr+='z'
        else:
            for i in range(n):
                strr+='p'
        return strr

File: LeetCode-Programs/Generate a String With Characters That Have Odd Counts.cpp
class Solution {
public:
    string generateTheString(int n) {
        string str="";
        if(n%2==0){
            for(int i=0;i<n-1;i++)
                str+='p';
            str+='p';
        }
        else{
            for(int i=0;i<n;i++)
                str+='z';
        }
        return str;
    }
};
